Assets/MixedRealityToolkit.Tests/PlayModeTests/Utils/TestButtonUtilities.cs

```diff
@@ -30,27 +30,27 @@ public enum DefaultButtonType
         };
 
         /// <summary>
-        /// Asset Path to default model push button prefab asset
+        /// Asset path to default model push button prefab asset
         /// </summary>
-        public const string DefaultInteractablePrefabAssetPath = "Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Prefabs/Model_PushButton.prefab";
+        public static readonly string DefaultInteractablePrefabAssetPath = AssetDatabase.GUIDToAssetPath(DefaultInteractablePrefabAssetGuid);
 
         /// <summary>
-        /// Asset path to default Hololens 2 Pressable Button prefab asset
+        /// Asset path to default HoloLens 2 Pressable Button prefab asset
         /// </summary>
-        public const string PressableHoloLens2PrefabPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButtonHoloLens2.prefab";
+        public static readonly string PressableHoloLens2PrefabPath = AssetDatabase.GUIDToAssetPath(PressableHoloLens2PrefabGuid);
 
         /// <summary>
-        /// Asset path to default Hololens 2 Toggle Pressable Button prefab asset
+        /// Asset path to default HoloLens 2 Toggle Pressable Button prefab asset
         /// </summary>
-        public const string PressableHoloLens2TogglePrefabPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButtonHoloLens2Toggle.prefab";
+        public static readonly string PressableHoloLens2TogglePrefabPath = AssetDatabase.GUIDToAssetPath(PressableHoloLens2TogglePrefabGuid);
```

Comment:
Realized there's a less invasive way to do this, which would be to add variables of this nature in each test class I updated. I'm gonna move to this pattern before merging